README.md

```diff
@@ -82,7 +82,22 @@ Read the [Plugin Developer Guide](./docs/DEVELOPER_GUIDE.md) for details.
 
 - [Architecture](./docs/KREW_ARCHITECTURE.md)
 - [Docs](./docs/)
-- [Contributing](./CONTRIBUTING.md)  
+- [Contributing](./CONTRIBUTING.md)
+
+# Roadmap
+
+## Support Multiple Index Repositories
```

Comment:
we can probably just make these like:

## Roadmap

- **Item 1:** explanation explanation explanation explanation explanation. 

     new paragraph explanation  explanation explanation 

- **Item 2:** ....

Reply:
I say this as it would make it more concise.